yoti/src/Controller/YotiStartController.php

```diff
@@ -89,4 +80,34 @@ public function binFile($field) {
     return new TrustedRedirectResponse('yoti.bin-file');
   }
 
+  /**
+   * Check if current account is linked.
+   *
+   * @param \Drupal\Core\Session\AccountInterface $account
+   *   Run access checks for this account.
+   *
+   * @return \Drupal\Core\Access\AccessResult
+   *   If account is linked isAllowed() will be TRUE, otherwise
+   *   isNeutral() will be TRUE.
+   */
+  public static function accessAccountIsLinked(AccountInterface $account) {
```

Comment:
Is the convention to prepend the method name with `access` if it's returning an AccessResult? Doesn't read that well to me, but maybe it's because I'm not used to it!

Reply:
I agree it doesn't read well - will change to `accessLink()` for access to `link()` and move `accessAccountIsLinked()` to `YotiUnlinkForm::access()`

Reply:
Fixed in 5c71394